modules/components/src/Admin/Tabs/DraggableListOrderer.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+
+import { toPairs, sortBy } from 'lodash';
+import { css } from 'emotion';
+import 'react-grid-layout/css/styles.css';
+import 'react-resizable/css/styles.css';
+import ReactGridLayout from 'react-grid-layout';
+import { fetchExtendedMapping } from '../../utils/api';
+
+class DraggableOrders extends React.Component {
+  state = {
+    rowHeight: 1, // normalizes to pixel level
+    layout: [],
+    extendedMapping: [],
+  };
+  listElements = {};
+
+  adjustHeight = async listElements =>
```

Comment:
Can you remove all of the resize logic and refs? just give everything a set height